1-js/04-object-basics/06-constructor-new/article.md

```diff
@@ -60,149 +60,149 @@ let user = {
-The syntax from this section is rarely used, skip it unless you want to know everything.
+```smart header="Fortgeschrittenes Thema"
+Die Syntax aus diesem Abschnitt wird selten verwendet, überspringen Sie ihn, es sei denn, Sie wollen alles wissen.
 ```
 
-Inside a function, we can check whether it was called with `new` or without it, using a special `new.target` property.
+Innerhalb einer Funktion können wir überprüfen, ob sie mit `new` oder ohne aufgerufen wurde, indem wir eine spezielle `new.target`-Eigenschaft verwenden.
 
-It is undefined for regular calls and equals the function if called with `new`:
+Sie ist undefiniert bei normalen Aufrufen und entspricht der Funktion, wenn sie mit `new` aufgerufen wird:
 
 ```js run
 function User() {
   alert(new.target);
 }
 
-// without "new":
+// ohne "new":
 *!*
 User(); // undefined
 */!*
 
-// with "new":
+// mit "new":
 *!*
 new User(); // function User { ... }
 */!*
 ```
 
-That can be used inside the function to know whether it was called with `new`, "in constructor mode", or without it, "in regular mode".
+Das kann innerhalb der Funktion verwendet werden, um herauszufinden, ob sie im "Konstruktormodus" mit `new` oder im "normalen Modus" ohne `new` aufgerufen wurde.
 
-We can also make both `new` and regular calls to do the same, like this:
+Wir können auch ermöglichen, dass sowohl `new` als auch normale Aufrufe das Gleiche tun:
 
 ```js run
 function User(name) {
-  if (!new.target) { // if you run me without new
-    return new User(name); // ...I will add new for you
+  if (!new.target) { // wenn Du mich ohne new aufrufst
+    return new User(name); // ... füge ich new für Dich hinzu
   }
 
   this.name = name;
 }
 
-let john = User("John"); // redirects call to new User
+let john = User("John"); // leitet den Aufruf zu new User um
 alert(john.name); // John
 ```
 
-This approach is sometimes used in libraries to make the syntax more flexible. So that people may call the function with or without `new`, and it still works.
+Dieser Ansatz wird manchmal in Bibliotheken verwendet, um die Syntax flexibler zu machen. So können Personen die Funktion mit oder ohne `new` aufrufen, und es funktioniert trotzdem.
 
-Probably not a good thing to use everywhere though, because omitting `new` makes it a bit less obvious what's going on. With `new` we all know that the new object is being created.
+Wahrscheinlich ist es aber keine gute Sache, diese überall zu verwenden, denn das Weglassen von `new` macht es etwas weniger offensichtlich, was vor sich geht. Mit `new` wissen wir alle, dass ein neues Objekt erstellt wird.
 
-## Return from constructors
+## Rückgabe aus Konstruktoren
 
-Usually, constructors do not have a `return` statement. Their task is to write all necessary stuff into `this`, and it automatically becomes the result.
+Normalerweise haben Konstruktoren keine `return`-Anweisung. Ihre Aufgabe ist es, alles Notwendige in `this` zu schreiben, und das wird automatisch zum Ergebnis.
```

Comment:
Du / Sie form

Reply:
Hier ist „Ihre“ keine Anrede sondern in der 3. Person, bezieht sich also auf Konstruktoren.